src/__experimental__/components/checkbox/checkbox.component.js

```diff
@@ -17,8 +18,16 @@ const Checkbox = ({
   labelHelp,
   labelSpacing = 1,
   ml,
+  adaptiveSpacingBreakpoint,
   ...props
 }) => {
+  const largeScreen = useIsAboveBreakpoint(adaptiveSpacingBreakpoint);
+
+  let marginLeft = ml;
+  if (adaptiveSpacingBreakpoint && !largeScreen) {
```

Comment:
When the user does not provide `adaptiveSpacingBreakpoint` should the hook return `false`?
If so we can change the condition to be 
```suggestion
  if (!largeScreen) {
```

Reply:
If we did that than `marginLeft` would always be set to '0' when you don't pass in a breakpoint

Reply:
We'd essentially be saying if there's no breakpoint set then it's always a small screen